### PR TITLE
Add GitHub Actions configure files

### PR DESCRIPTION
**Pull request by conao3**
This replaces #153.

Add CI via GitHub Actions.
This CI working my fork [like that](https://github.com/conao3/helm-swoop/commit/701c0f33b1045e70a8233b4981ca965ac5f6c887/checks?check_suite_id=372942469).

### CODE REVIEW

File: Cask [hunk 1]
```diff
@@ -0,0 +1,9 @@
+;; -*- mode:lisp -*-
+
+(source gnu)
+(source melpa)
+
+(package-file "helm-swoop.el")
+
+(development
```
Comment (cireu) [inline Cask]: You hadn't add `helm` as dependency :joy:

Comment (conao3) [reply Cask]: Needless. Cask read dependency from the header of helm-swoop.el.
One piece of information should place one place.

Comment (cireu) [reply Cask]: That's really good.

**Comment by cireu** [comment]
LGTM

**Comment by conao3** [comment]
Thanks! merge.

**Pull request merged**